File: beginner/pagina12/3170/3170.cpp
#include <iostream>

using namespace std;

int main(){

    int b, g;
    cin >> b >> g;

    int result = (g/2) - b;

    if(result <= 0){
        cout << "Amelia tem todas bolinhas!" << endl;
    } else {
        cout << "Faltam " << result << " bolinha(s)!" << endl;
    }

    return 0;
}

File: beginner/pagina12/3170/3170.js
const fs = require('fs')
const input = fs.readFileSync('/dev/stdin', 'utf8');
const [b, g] = input.split('\n').map(Number)

const result = parseInt((g /2) - b)

console.log(result <= 0 ? "Amelia tem todas bolinhas!" : `Faltam ${result} bolinha(s)!`)